website/docs/notes/concepts.md

```diff
@@ -21,52 +19,71 @@ To achieve this, MMF has few opinions about architecture of your research projec
 
 In MMF, we have divided datasets into a set category of tasks. Thus, a task corresponds to a collection of datasets that belong to it. For example, VQA 2.0, VizWiz and TextVQA all belong VQA task. Each task and dataset has been assigned a unique key which is used to refer it in the command line arguments.
 
-Following table shows the tasks and their datasets:
-
-|         |            |                                              |
-| ------- | ---------- | -------------------------------------------- |
-| Task    | Key        | Datasets                                     |
-| VQA     | vqa        | VQA2.0, VizWiz, TextVQA, VisualGenome, CLEVR |
-| Dialog  | dialog     | VisualDialog                                 |
-| Caption | captioning | MS COCO                                      |
+Following table shows the example tasks and their datasets:
+
+| Task             | Key        | Datasets                                                      |
+| ---------------- | ---------- | --------------------------------------------------------------|
+| VQA              | vqa        | VQA2.0, VizWiz, TextVQA, VisualGenome, CLEVR, OKVQA, GQA      |
+| Dialog           | dialog     | VisualDialog                                                  |
+| Caption          | captioning | MS COCO, Conceptual Captions, Localized Narratives, Flickr30k |
+
+The following table shows a complete set of datasets available in mmf,
```

Comment:
This replicates the list of dataset available in docs. Let's point there instead.